staging/src/k8s.io/apiserver/pkg/server/deprecated_insecure_serving.go

```diff
@@ -50,7 +50,8 @@ func (s *DeprecatedInsecureServingInfo) Serve(handler http.Handler, shutdownTime
 	} else {
 		klog.Infof("Serving insecurely on %s", s.Listener.Addr())
 	}
-	return RunServer(insecureServer, s.Listener, shutdownTimeout, stopCh)
+	_, err := RunServer(insecureServer, s.Listener, shutdownTimeout, stopCh)
```

Comment:
does discarding the returned channel mean we won't wait for requests to the unsecured port to complete?

Reply:
yes, that's the case. We promised to remove the insecure port long time ago. Maybe we should finally do it.